build/journalbeat/config/journalbeat.yml

```diff
@@ -0,0 +1,11 @@
+journalbeat.inputs:
+- paths: [/mnt/journal]
+
+processors:
+  - add_host_metadata: ~
+  - add_cloud_metadata: ~
+
```

Comment:
I notice that we use `~` in the Beats documentation too, but isn't `null` a lot more obvious to people who don't keep the YAML spec under their pillow?

Reply:
It might be more obvious, but I would like to follow the "patterns" used in the documentation to stay consistent.

Reply:
I guess I was politely hinting that we could improve the docs. I had to refer to the YAML spec to figure out what `~` meant, while `null` would have been instantly understandable.

Anyway, I'll leave it up to you and the Beats team.

cc @exekias

Reply:
I see. I try to be consistent with the docs and other config examples to help users understand how to use Beats. I am not an authority on Beats documentation. That is @dedemorton 
If we change `~` to `null` consistently everywhere, I am happy to change these two instances to `null`.

Reply:
Maybe for coherence with [other config files](https://github.com/elastic/beats-docker/blob/37d440b1166d4062c7e17d6ae68d70612175bcf8/build/filebeat/config/filebeat.yml) in this repository we can leave these as enmpty values:
```
processors:
  - add_host_metadata:
  - add_cloud_metadata:
```

In any case I would leave the introduction of `null` for future discussions and PRs as we are not using it anywhere.

Reply:
I'm ok with switching this to @jsoriano proposal, if we do, it would be interesting to update reference + default files too (outside docker)

Reply:
Donz.

Reply:
Thanks, everyone!